addons/account/models/chart_template.py

```diff
@@ -115,6 +115,7 @@ class AccountChartTemplate(models.Model):
         string="Gain Exchange Rate Account", domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)])
     expense_currency_exchange_account_id = fields.Many2one('account.account.template',
         string="Loss Exchange Rate Account", domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)])
+    default_pos_receivable_account_id = fields.Many2one('account.account.template', string="PoS receivable account")
```

Comment:
you can manage the property like in the PR of rhe: https://github.com/odoo/odoo/pull/33948/files#diff-16a531be157c0d2d6c146ff7cc53413bR118

Reply:
Making this field company dependent is not a good idea, IMO, because of the cash_journal_id field. Both fields should belong in the same company. if receivable_account_id becomes company dependent, when the user changed company, the field's value changes but the cash_journal_id field does not. Making the whole pos.payment.method record to contain inconsistent fields.

Reply:
I think making this a company property is better than making it a property for the account.journal. Besides, account.journal has less to do with the pos.payment.method. In fact, the cash_journal_id field is only required when is_cash_count is active. I don't understand why I should make this property of account.journal.

Reply:
I've just checked with AL, and you are right, we should set a field, which is a default on the company and use it as default and if not don't set any. (The first version :D )